resourcemanager/resourcemanager.go

```diff
@@ -84,6 +84,33 @@ func (rm *ResourceManager) UpdateResources() {
 	}
 }
 
+func (rm *ResourceManager) RemovedInvalidUIDs() {
+	resources := rm.store.GetResources()
+
+	for _, r := range resources {
+		members := rm.store.GetMembers()
+		for _, m := range members {
+			if m.Level != uint8(models.Inactive) {
+				return
+			}
+
+			if len(m.RFID) == 0 {
+				return
+			}
+
+			/* We will just try to remove all invalid members even if they are already removed */
+			b, _ := json.Marshal(&models.AddMemberRequest{
```

Comment:
Shouldn't be `DeleteMemberRequest`? I find it weird then we are removing by using the AddMemberRequest